server/schema/routes.sql

```diff
@@ -1,4 +1,4 @@
 CREATE TABLE routes (
   id SERIAL PRIMARY KEY,
-  driver_id VARCHAR(256)
-);
\ No newline at end of file
+  driver_id VARCHAR(256) REFERENCES users(id) ON DELETE CASCADE
```

Comment:
If a driver gets deleted, all routes associated with that driver should not be deleted. The exact action is still to be determined, so for now, change the `on delete cascade` to `no action`.